src/sdl2/audio.rs

```diff
@@ -381,9 +382,10 @@ pub struct AudioSpecDesired {
 }
 
 impl AudioSpecDesired {
-    fn convert_to_ll<CB: AudioCallback>(freq: Option<i32>, channels: Option<u8>, samples: Option<u16>, userdata: *mut CB) -> ll::SDL_AudioSpec {
+    fn convert_to_ll<CB: AudioCallback, FREQ: Into<Option<i32>>,>(freq: FREQ, channels: Option<u8>, samples: Option<u16>, userdata: *mut CB) -> ll::SDL_AudioSpec {
```

Comment:
Interestingly you converted Freq into a `Into<_><è, but left out `channels` and `samples` here, is there a reason ?